llvm/utils/TableGen/Basic/DirectiveEmitter.cpp

```diff
@@ -642,7 +642,8 @@ static void emitLeafTable(const DirectiveLanguage &DirLang, raw_ostream &OS,
     auto &LeavesB = LeafTable[B];
     int DirA = LeavesA[0], DirB = LeavesB[0];
     // First of all, end directives compare greater than non-end directives.
-    int IsEndA = EndDirectives.count(DirA), IsEndB = EndDirectives.count(DirB);
+    int IsEndA = EndDirectives.contains(DirA);
```

Comment:
nit: make these bool?

Reply:
I think in C++ < operator for bools is defined the natural way: Google search AI overview:

> This behavior is consistent with the numerical interpretation of false and true as 0 and 1, respectively